app/src/main/java/com/duycuong/weather/ui/screen/main/current_weather/CurrentWeatherFragment.java

```diff
@@ -0,0 +1,36 @@
+package com.duycuong.weather.ui.screen.main.current_weather;
+
+import android.location.Location;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports